src/screens/EditProfile/EditProfile.jsx

```diff
@@ -70,7 +80,7 @@ const AboutUser = () => {
 
   return (
     <div className="about-user">
-      <Form formData={userForm} formState={[userInfo, setUserInfo, handleUserUpdate]} />
+      <EditProfileForm formData={userForm} formState={[userInfo, setUserInfo, handleUserUpdate]} />
```

Comment:
Let's relieve ourselves from the need of a custom-input-vending component. A form from scratch is going to be much easier to go back and update and ultimately gives us more control vs. a headache.

Don't build another component, you can build the form directly in here.